### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 11

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#187] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152253037-04d2a718-450b-4313-8eb0-b87932b71ac2.png)

**Comment by Exyss** [comment]
$\binom{12}{3} \binom{9}{3} \binom{6}{3}$

dunque 3 persone rimangono in piedi, visto che abbiamo solo 3 tavoli

_Stesso approccio risolutivo usato per gli esercizi #8, #11_

**Reply by yasuoCode** [reply]
Credo che alla soluzione vada aggiunta una divisione per $3!$ per scongiurare l'ordine poiché l'assegnazione del gruppo $A$ al primo tavolo e del gruppo $B$ al secondo è la stessa dell'assegnazione del gruppo $B$ al primo tavolo e del gruppo $A$ al secondo.

Quindi la soluzione diventa:

$$\frac {\binom{12}{3}\cdot \binom{9}{3}\cdot \binom{6}{3}}{3!}$$

Edit: [Simile all'esercizio 10](https://github.com/sapienzastudentsnetwork/metodi-matematici-per-l-informatica/discussions/186#discussioncomment-11864093)
